medusa/name_parser/rules/rules.py

```diff
@@ -1220,8 +1222,8 @@ def when(self, matches, context):
                     to_remove.extend(matches.named('title', predicate=lambda match: match.value != values[0].value))
                     continue
 
-                logger.info(u"Guessed more than one '%s' for '%s': %s",
-                            name, matches.input_string, u','.join(unique_values), exc_info=False)
+                log.info(u"Guessed more than one '%s' for '%s': %s",
```

Comment:
@p0psicles I left this unchanged intentionally, as there was only a single log line I figured that importing the `BraceAdapter` may not be necessary.  Thoughts?